### PR TITLE
Github Workflow for automated .deb package build

### PR DESCRIPTION
**Pull request by brs-brs**
This pull request introduces a GitHub Workflow for automating the build process of .deb packages. This enhancement aims to streamline the usage of xkb-switch for users on Debian-based systems, making it more convenient and accessible.

### CODE REVIEW

File: .github/workflows/deb-build.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      run: |
+        mkdir -p build
+        cd build
+        cmake ..
+        make
+
+    - name: Package project
+      id: package
+      run: |
+        cd build
```
Comment (sergei-mironov) [inline .github/workflows/deb-build.yml:45]: I'm skeptical about tightening the project with the github-ci infrastructure, although I do appreciate the benefits of having latest .deb artifacts, even unchecked and unsigned. Could you please move `cpack` to a CMake rule and just call this rule from here?

File: man/xkb-switch.1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.TH "xkb-switch" "1" "1.3.1" "J. Bromley, S. Mironov, Alexei Rad'kov" "User Commands"
+.TH "xkb-switch" "1" "2.1.1" "J. Bromley, S. Mironov, Alexei Rad'kov" "User Commands"
```
Comment (sergei-mironov) [inline man/xkb-switch.1:1]: Good catch, thanks!

File: src/XKeyboard.cpp [hunk 3]
```diff
@@ -196,6 +196,37 @@ int XKeyboard::get_group() const
...
+  if (XkbGetState(_display, _deviceId, &xkbState) != Success) {
+    throw std::runtime_error("Failed to get keyboard state.");
+  }
+
+  XkbDescPtr desc = XkbGetKeyboard(_display, XkbAllComponentsMask, _deviceId);
+  if (desc == nullptr) {
+    throw std::runtime_error("Failed to get keyboard description.");
+  }
+
+  char* groupName = XGetAtomName(_display, desc->names->groups[xkbState.group]);
```
Comment (sergei-mironov) [inline src/XKeyboard.cpp:216]: Could you please check the lengths before accessing this array element?

**Review by sergei-mironov — CHANGES_REQUESTED** [review]
Thanks for your work! I have just a few minor comments.

**Comment by brs-brs** [comment]
Hi @sergei-mironov ! Now requested changes can be reviewed

**Review by sergei-mironov — APPROVED** [review]
Thanks, @brs-brs ! I will merge this PR into a branch and make some final adjustments

**Pull request merged**